### ISSUE TITLE
normalize() maybe wrong?

### ISSUE THREAD

**Issue by freedom12** [nicklockwood/VectorMath#21]
```
func normalized() -> Vector4 {
    let lengthSquared = self.lengthSquared
    if lengthSquared ~= 0 || lengthSquared ~= 1 {
        return self
    }
    return self / sqrt(lengthSquared)
}

```

should be " lengthSquared == 0 || lengthSquared == 1 "

**Comment by nicklockwood** [comment]
Why?

**Comment by nicklockwood** [comment]
Oh, hmm I see. I think the problem is really that the value of epsilon is way too large. Should probably much closer to the actual limit of float precision.

**Comment by nicklockwood** [comment]
Fixed in 0.4.1
